AddressParser/AddressParser.cs

```diff
@@ -928,6 +937,33 @@ public AddressParseResult ParseAddress(string input, bool normalize)
             return new AddressParseResult(extracted);
         }
 
+        public AddressParseResult ParseAddressLine(string input, bool normalize)
```

Comment:
This is all duplicated code that can be deduplicated by creating the `ParseAddress` overload I've suggested multiple times.  The overload is *going* to be the final design.  If you do it then you save work for me, otherwise, I have to refactor this after accepting the PR.  So please just save me the work and create the overload.